docs/advanced-usage/satellite-domains.mdx

```diff
@@ -105,23 +105,21 @@ The `URL` mentioned above is the request url for server side usage or the curren
     ```
 
     ```js filename="middleware.ts"
-    import { authMiddleware, redirectToSignIn } from "@clerk/nextjs/server";
-
-    export default authMiddleware({
-      afterAuth(auth) {
-        // handle users who aren't authenticated
-        if (!auth.userId && !auth.isPublicRoute) {
-          return redirectToSignIn({ returnBackUrl: "https://satellite.dev" });
-        }
-      },
-      isSatellite: true,
-      signInUrl: 'https://primary.dev/sign-in'
-      domain: url => url.host
```

Comment:
Are we able to cut all these extra options here and still have it work correctly?

Reply:
I'm not either - may be worth asking somewhere before we ship

Reply:
OHHH I see what you mean - my edits cut these options out. I did *not* mean to do that, I specifically meant to keep those in